e2e/tests/customPages.js

```diff
@@ -19,7 +19,8 @@ describe('Custom Pages', () => {
         });
 
         it('should have retrieved the number of posts', async () => {
-            assert.equal(await CustomPageNoLayout.getTotal(), '13');
+            // 14 because we created one in custom-forms tests
```

Comment:
I'd prefer if you deleted the record you created. We should avoid side effects in e2e tests.